examples/batch_logging.cc

```diff
@@ -45,16 +46,16 @@ void JobLogs(std::vector<std::string> const& argv) {
   // [START batch_job_logs]
   [](std::string const& project_id, std::string const& location_id,
      std::string const& job_id) {
-    auto const name = "projects/" + project_id + "/locations/" + location_id +
-                      "/jobs/" + job_id;
+    auto const project = google::cloud::Project(project_id);
+    auto const location = google::cloud::Location(project, location_id);
+    auto const name = location.FullName() + "/jobs/" + job_id;
     auto batch = google::cloud::batch_v1::BatchServiceClient(
         google::cloud::batch_v1::MakeBatchServiceConnection());
     auto job = batch.GetJob(name);
     if (!job) throw std::move(job).status();
 
     auto logging = google::cloud::logging_v2::LoggingServiceV2Client(
         google::cloud::logging_v2::MakeLoggingServiceV2Connection());
-    auto const project = google::cloud::Project(project_id);
     auto const log_name = "projects/" + project_id + "/logs/batch_task_logs";
```

Comment:
Should we use the `project.FullName()` here too?

Reply:
Done.